app/views.py

```diff
@@ -87,8 +87,19 @@ def login():
             return "There was an error.", 500
         # Else success, redirect to index page
         else:
-            session['username'] = username
             session['token'] = token['token']
+            user = getUser(username)
+
+            if type(user) is list:
```

Comment:
@ElijahCaine It should return the user object, so we can store it in the session.